descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "invoice",
-      "version": "5.6",
+      "version": "6.0",
```

Comment:
Don't forget to update `invoice-storage.invoices` version in requires section

Reply:
Done